examples/ReadSensorsImperial/ReadSensorsImperial.ino

```diff
@@ -0,0 +1,68 @@
+/*
+  MKR ENV Shield - Read Sensors Imperial
+
+  This example reads the sensors on-board the MKR ENV shield
+  and prints them in imeprial units to the Serial Monitor once a second.
+
+  The circuit:
+  - Arduino MKR board
+  - Arduino MKR ENV Shield attached
+
+  This example code is in the public domain.
+*/
+
+#include <Arduino_MKRENV.h>
+
+void setup() {
+  Serial.begin(9600);
+  while (!Serial);
+
+  if (!ENV.begin()) {
+    Serial.println("Failed to initialize MKR ENV shield!");
+    while (1);
+  }
+}
+
+void loop() {
+  // Seting to FAHRENHEIT and FOOTCANDLE readTemperature and readIlluminance
+  // allow to reads in imperial units the temperature and illuminance sensor 
+  float temperature = ENV.readTemperature(FAHRENHEIT);
+  float humidity    = ENV.readHumidity();
+  float pressure    = ENV.readPressure();
+  float illuminance = ENV.readIlluminance(FOOTCANDLE);
+  float uva         = ENV.readUVA();
+  float uvb         = ENV.readUVB();
+  float uvIndex     = ENV.readUVIndex();
+
+  // print each of the sensor values
+  Serial.print("Temperature = ");
+  Serial.print(temperature);
+  Serial.println(" °F");
+
+  Serial.print("Humidity    = ");
+  Serial.print(humidity);
+  Serial.println(" %");
+
+  Serial.print("Pressure    = ");
+  Serial.print(pressure);
+  Serial.println(" kPa");
+
+  Serial.print("Illuminance = ");
+  Serial.print(illuminance);
+  Serial.println(" mc");
```

Comment:
Shouldn't the abbreviation here be `fc`?

https://en.wikipedia.org/wiki/Foot-candle

Reply:
done